stylelint.config.js

```diff
@@ -1,7 +1,11 @@
-// BEM Selector Class Configs
-const customBemSelector = () => {
+// Transform from PascalCase to kebab-case
+const pascalToKebabCase = str => str.replace(/\.?([A-Z])/g, (char) =>  "-" + char.toLowerCase()).replace(/^-/, "");
```

Comment:
Is this supposed to be in here?

Reply:
Nope, I fixed the commit history so this shouldn't be in this PR anymore.